lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
> I get the point that the type: 'text' on every node is basically JSON HTML, but it seems like it's valuing simplicity in the report renderer at the expense of more verbose and less readable JSON results

Now that I look at the typedef, I'm even more curious about the thinking behind the `details` object. `/** @typedef {{type: string, text: string|undefined, header: DetailsJSON|undefined, items: Array<DetailsJSON>|undefined}} */` seems like an overgeneralization. It's cool you could render a list in the header of a list, but...why? Seems better to make a new `type` at that point, otherwise a reader will just wonder what on earth is going on.